src/test/java/springbook/inmemory/EmbeddedDbTest.java

```diff
@@ -0,0 +1,45 @@
+package springbook.inmemory;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Map;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.jdbc.datasource.embedded.EmbeddedDatabase;
+import org.springframework.jdbc.datasource.embedded.EmbeddedDatabaseBuilder;
+import org.springframework.jdbc.datasource.embedded.EmbeddedDatabaseType;
+
+public class EmbeddedDbTest {
+
+    private EmbeddedDatabase db;
+    private NamedParameterJdbcTemplate template;
+
+    @BeforeEach
+    void setUp() {
+        db = new EmbeddedDatabaseBuilder()
+                .setType(EmbeddedDatabaseType.H2)
+                .addScript("init.sql")
+                .build();
+
+        template = new NamedParameterJdbcTemplate(db);
```

Comment:
### ❗ deprecated된 SimpleJdbcTemplate
책에서는 SimpleJdbcTemplate를 사용했으나 해당 클래스 대신 NamedParamenterJdbcTemplate을 사용하였다.

> Deprecated. since Spring 3.1 in favor of [JdbcTemplate](https://docs.spring.io/spring-framework/docs/3.1.x/javadoc-api/org/springframework/jdbc/core/JdbcTemplate.html) and [NamedParameterJdbcTemplate](https://docs.spring.io/spring-framework/docs/3.1.x/javadoc-api/org/springframework/jdbc/core/namedparam/NamedParameterJdbcTemplate.html). The JdbcTemplate and NamedParameterJdbcTemplate now provide all the functionality of the SimpleJdbcTemplate.

출처: [Spring Docs](https://docs.spring.io/spring-framework/docs/3.1.x/javadoc-api/org/springframework/jdbc/core/simple/SimpleJdbcTemplate.html)